extra/2-piping.js

```diff
@@ -16,26 +16,29 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  return num1 * num2;
 }
 
-function format() {
-
+function format(number) {
+  return "£" + number;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+// It is hard to read
+let badCode = add(startingValue, 10); badCode = multiply(badCode, 2); badCode = format(badCode);
 
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = add(startingValue, 10);
```

Comment:
Good work on completing the additional exercise as well Anthony.
You may also consider combining all the function calls into one expression as follows:
`let goodCode = format(multiply(add(startingValue, 10), 2));`